<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>K6GTE winkeyerserial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Sent Text</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="inputbox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>153</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_speed">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_device">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="outputbox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Free text input</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="sendmsg1_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="sendmsg2_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="sendmsg3_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 3</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="sendmsg4_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 4</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="sendmsg5_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 5</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="sendmsg6_button">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="text">
       <string>msg 6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="msg1">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="msg2">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="msg3">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLineEdit" name="msg4">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLineEdit" name="msg5">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLineEdit" name="msg6">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
